modules/sample/src/main/resources/petstore.json

```diff
@@ -579,6 +579,9 @@
         "summary": "Place an order for a pet",
         "description": "",
         "operationId": "placeOrder",
+        "consumes": [
```

Comment:
I agree the previous behavior was broken, though changing the default should probably be done separate from the bugfix, with a reasonable deprecation cycle, permitting folk to migrate without being forced to react without notice.

To clarify, these extra `consumes` are not explicitly required, correct?

Reply:
I'm not sure what you are suggesting. Can you elaborate more what you have in mind?

Regarding extra `consumes`: it's not required, but if it is missing a user is required to provide `EntityDecoder` (server), or `EntityEncoder` (client) explicitly (using some sort of import).

Reply:
Yikes! Previously it defaulted to `application/json` though, unless that was what was previously broken? Could we not just assume JSON as a fallback if the field is empty? You're always welcome (and encouraged!) to provide that field, and possibly in the future we could make an empty `produces`/`consumes` an error on codegen, but that's what I'm suggesting needs a deprecation cycle (warn that behavior is changing, then make it an error eventually)

Reply:
I think that previously it was meant to default to application/json, but the implementation of that was completely broken.

It should be pretty easy to default to `application/json` though. What I'm saying is that it makes no sense to me to deprecate behavior that never worked in the first place. But I'll be happy to implement a behavior that defaults to `application/json` (perhaps with some warning message).

Reply:
Ah, that clarifies your position. Considering that the default for the other implementations (clients/servers across languages) is `application/json`, it would be good to default to that here as well. If you'd like to add a warning, the place to do it is in guardrail core, instead of having disparate behaviour in the individual framework interpreters. Depends on how strongly you feel about it though.